nav2_system_tests/params/keepout_params.yaml

```diff
@@ -1,7 +1,306 @@
+amcl:
+  ros__parameters:
+    use_sim_time: True
+    alpha1: 0.2
+    alpha2: 0.2
+    alpha3: 0.2
+    alpha4: 0.2
+    alpha5: 0.2
+    base_frame_id: "base_footprint"
+    beam_skip_distance: 0.5
+    beam_skip_error_threshold: 0.9
+    beam_skip_threshold: 0.3
+    do_beamskip: false
+    global_frame_id: "map"
+    lambda_short: 0.1
+    laser_likelihood_max_dist: 2.0
+    laser_max_range: 100.0
+    laser_min_range: -1.0
+    laser_model_type: "likelihood_field"
```

Comment:
Change this to another laser model, that will push up AMCL's smoke coverage.

Reply:
Do you mean `likelihood_field_prob` ?

Reply:
yeah, `likelihood_field_prob` would be fine. Or `beam`, I don't really care which. Its just about smoke testing another model if we're specifying a custom param yaml file.